src/main/java/com/nashtech/JavaCompetencyDemoApplication.java

```diff
@@ -4,6 +4,8 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
+@EnableReactiveCosmosRepositories
+(basePackages = "com.nashtech.repository")
 public class JavaCompetencyDemoApplication {
```

Comment:
Rename the class as per your project. JavaCompetencyDemoApplication does not tell what the application is all about.